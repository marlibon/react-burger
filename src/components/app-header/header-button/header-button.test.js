import React from 'react';
import renderer from 'react-test-renderer';

import HeaderButton from './header-button';
it('render app', () => {
  const tree = renderer
    .create(<HeaderButton children={null} text="text" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
