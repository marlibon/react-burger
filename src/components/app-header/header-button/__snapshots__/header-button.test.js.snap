// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<button
  className="button mr-2"
  type="button"
>
  <span
    className="text text_type_main-default"
  >
    text
  </span>
</button>
`;
